omegaconf/nodes.py

```diff
@@ -335,16 +337,13 @@ def __init__(
             ),
         )
 
-    def validate_and_convert(self, value: Any) -> Optional[Enum]:
+    def _validate_and_convert_impl(self, value: Any) -> Optional[Enum]:
         return self.validate_and_convert_to_enum(enum_type=self.enum_type, value=value)
 
     @staticmethod
     def validate_and_convert_to_enum(
-        enum_type: Type[Enum], value: Any, allow_none: bool = True
+        enum_type: Type[Enum], value: Any
     ) -> Optional[Enum]:
-        if allow_none and value is None:
-            return None
-
```

Comment:
Can the return type annotation for this function be changed from `Optional[Enum]` to `Enum`?

Reply:
Yes that's a good point, done in cd8655b